hera_qm/ant_class.py

```diff
@@ -320,6 +321,60 @@ def auto_slope_checker(data, good=(-.2, .2), suspect=(-.4, .4), edge_cut=100, fi
 
     return antenna_bounds_checker(relative_slopes, good=good, suspect=suspect, bad=(-np.inf, np.inf))
 
+def auto_rfi_checker(data, good=(0, 0.1), suspect=(0.1, 0.2), nsig=6, antenna_class=None, flag_threshold=0.1, 
+                     kernel_widths=[3, 4, 5], mode='dpss_matrix', filter_centers=[0, -2700e-9, 2700e-9],
```

Comment:
These filter centers are a bit odd and really specialized to the H4C system. Do we actually need to include the 2700 ns feature?

Reply:
Changed this. We can add those options as input as needed